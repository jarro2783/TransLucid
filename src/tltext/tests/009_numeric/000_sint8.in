%%
sint8"0";;
sint8"1";;
sint8"12";;
sint8"127";;
sint8"-128";;

sint8"0" + sint8"0";;
sint8"15" + sint8"0";;
sint8"0" + sint8"15";;
sint8"8" + sint8"5";;
sint8"-128" + sint8"130";;
sint8"-128" + sint8"5";;

sint8"0" - sint8"0";;
sint8"10" - sint8"0";;
sint8"0" - sint8"10";;
sint8"15" - sint8"3";;

sint8"6" * sint8"1";;
sint8"6" * sint8"0";;
sint8"-5" * sint8"4";;
sint8"-5" * sint8"-4";;

sint8"5" / sint8"1";;
sint8"23" / sint8"5";;
sint8"-10" / sint8"2";;

sint8"7" % sint8"4";;

sint8"0" < sint8"1";;
sint8"1" < sint8"1";;
sint8"2" < sint8"1";;

sint8"0" <= sint8"1";;
sint8"1" <= sint8"1";;
sint8"2" <= sint8"1";;

sint8"0" > sint8"1";;
sint8"1" > sint8"1";;
sint8"2" > sint8"1";;

sint8"0" >= sint8"1";;
sint8"1" >= sint8"1";;
sint8"2" >= sint8"1";;

sint8"0" == sint8"1";;
sint8"1" == sint8"1";;
sint8"2" == sint8"1";;

sint8"0" != sint8"1";;
sint8"1" != sint8"1";;
sint8"2" != sint8"1";;

sint8"5"!prec;;
sint8"5"!is_signed;;
