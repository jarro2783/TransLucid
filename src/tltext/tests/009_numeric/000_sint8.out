sint8"0"
sint8"1"
sint8"12"
sint8"127"
sint8"-128"
sint8"0"
sint8"15"
sint8"15"
sint8"13"
sint8"2"
sint8"-123"
sint8"0"
sint8"10"
sint8"-10"
sint8"12"
sint8"6"
sint8"0"
sint8"-20"
sint8"20"
sint8"5"
sint8"4"
sint8"-5"
sint8"3"
true
false
false
true
true
false
false
false
true
false
true
true
false
true
false
true
false
true
8
true
